common/app/conf/switches/PerformanceSwitches.scala

```diff
@@ -148,8 +148,8 @@ trait PerformanceSwitches {
 
   val DiscussionSwitch = Switch(
     SwitchGroup.Performance,
-    "discussion",
-    "If this switch is on, comments are displayed on articles. Turn this off if the Discussion API is blowing up.",
+    "comments-visible-on-article",
+    "If this switch is on, comments are displayed on articles on dotcom. This switch only show/hides the UI from the articles on dotcom and does not disable any underlying services/endpoints etc..",
```

Comment:
One more nit. This only hides it from the bottom of the articles. It doesn't do anything else I don't think. Probably best to make that clear.